sound/soc/sof/ipc4-topology.c

```diff
@@ -882,10 +882,8 @@ static int sof_ipc4_widget_setup_comp_process(struct snd_sof_widget *swidget)
 	/* allocate memory for base config extension if needed */
 	if (process->init_config == SOF_IPC4_MODULE_INIT_CONFIG_TYPE_BASE_CFG_WITH_EXT) {
 		struct sof_ipc4_base_module_cfg_ext *base_cfg_ext;
-		u32 ext_size;
-
-		ext_size += struct_size(base_cfg_ext, pin_formats,
-					swidget->num_input_pins + swidget->num_output_pins);
+		u32 ext_size = struct_size(base_cfg_ext, pin_formats,
+						swidget->num_input_pins + swidget->num_output_pins);
```

Comment:
I think the commit title needs to be 

fixup! ASoC: SOF: ipc4-topology: Add support for base config extension

and the Fixes tag will be dropped as well, we only use it for patches that have landed in Mark Brown's tree.

Reply:
Ah I didnt check marks, will update title

Reply:
@cujomalainey can you please try ```git commit --fixup``` with the hash for the commit in Pierre's comment.

Reply:
Did not know that was a command, thanks for the pointer

Reply:
@cujomalainey, can you update the commit title to:
`fixup! ASoC: SOF: ipc4-topology: Add support for base config extension`
so we will know which patch this should be applied to?

I wonder how things worked with this bug... Thanks for finding it!!!